### PR TITLE
[Build] remove cmake

### PR DESCRIPTION
**Pull request by jaeyun-jung**
transition to meson, remove all the cmake script.

relate issue : #927

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

**Comment by myungjoo** [comment]
PTAL: @jijoongmoon @leemgs @again4you @wooksong @helloahn This may affect your developmental environment. This is also another final change for 0.1.0 release.

File: Documentation/getting-started.md [hunk 1]
```diff
@@ -28,59 +28,39 @@ If there is a missing package, debuild will tell you which package is missing.
...
-
-**Approach 3.** Build with meson
+**Approach 2.** Build with meson
 * https://mesonbuild.com/Getting-meson.html
 
-install the required packages.
+Install the required packages. (meson >= 0.40)
 
-```bash
+```
```
Comment (leemgs) [inline Documentation/getting-started.md]: @jaeyun-jung  Why do you  want to replace "```bash" with "```"?  In my case, I much prefer the existing style to the change.

Comment (jaeyun-jung) [reply Documentation/getting-started.md]: I thought the optional language identifier is unnecessary.

Comment (leemgs) [reply Documentation/getting-started.md]: Let's keep the existing style because the statements is shell command.

Comment (jaeyun-jung) [reply Documentation/getting-started.md]: Now re-updated with identifier.

**Pull request merged**